.expeditor/buildkite/URL-verification.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eou pipefail
+
+TARGET_ENVIRONMENT="${ENVIRONMENT:-acceptance}"
+
+# This block tests the "environment" urls
+if [ "$TARGET_ENVIRONMENT" == "acceptance" ]; then
+  curl --fail -I "https://omnitruck-acceptance.chef.io/_status"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/install.sh"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/install.ps1"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/stable/chef/versions"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/stable/chef/metadata?p=ubuntu&pv=18.04&m=x86_64"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/stable/chef/download?p=ubuntu&pv=18.04&m=x86_64"
+elif [ "$TARGET_ENVIRONMENT" == "production" ]; then
+  curl --fail -I "https://omnitruck.chef.io/_status"
+  curl --fail -I "https://omnitruck.chef.io/install.sh"
+  curl --fail -I "https://omnitruck.chef.io/install.ps1"
+  curl --fail -I "https://omnitruck.chef.io/stable/chef/versions"
+  curl --fail -I "https://omnitruck.chef.io/stable/chef/metadata?p=ubuntu&pv=18.04&m=x86_64"
+  curl --fail -I "https://omnitruck.chef.io/stable/chef/download?p=ubuntu&pv=18.04&m=x86_64"
+else
```

Comment:
We should DRY this up by just using the if/else to select the target domain (e.g. `omnitruck-acceptance.chef.io` and `omnitruck.chef.io`) which can be used in one set of the curl commands...something like:
```
curl --fail -I "https://$TARGET_DOMAIN/_status"
```